src/gridcoin/staking/reward.cpp

```diff
@@ -46,7 +46,12 @@ CAmount GRC::GetConstantBlockReward(const CBlockIndex* index)
     //TODO: refactor the expire checking to subroutine
     //Note: time constant is same as GetBeaconPublicKey
     if ((index->nTime - oCBReward.timestamp) <= (60 * 24 * 30 * 6 * 60)) {
-        reward = atoi64(oCBReward.value);
+        // This is a little slippery, because if we ever change CAmount from a int64_t, this will
+        // break. It is unlikely to ever change, however, and avoids an extra copy/implicit cast.
+        if (!ParseInt64(oCBReward.value, &reward)) {
```

Comment:
This is a slight behavior change. If CBR is missing in the cache the old implementation would set `reward` to 0 whereas the new sets it to / leaves it at `DEFAULT_CBR`.